Source/IntegrationTests/TestFiles/LitTests/LitTest/cli/projectFile/libs/projectAsLibrary.dfy

```diff
@@ -0,0 +1,12 @@
+// RUN: %resolve --library "%S/dfyconfig.toml" --library %S/wrappersLib.dfy --allow-warnings --function-syntax 3 %s > "%t"
```

Comment:
We have to pass `--library %S/wrappersLib.dfy` as well because it' a dependency of our dependency.